### ISSUE TITLE
Confirm compatibiliy with node 16

### ISSUE THREAD

**Issue by bidoubiwa** [meilisearch/instant-meilisearch#428]
**Description**
Node 16 is now the latest version of node.  It is not the `active` version yet so we should not proactively confirm compatibility with node 16. Nonetheless, the 26 October 2021 node 16 will become the active version and this is a reminder for that we have to guarantee compatibility.

[See node releases status](https://nodejs.org/en/about/releases/)

**Environment (please complete the following information):**
- OS: IOS
- MeiliSearch version: v.0.20.0
- docs-searchbar.js version: v1.1.11

**Comment by sanders41** [comment]
I have Node v16.1.0 installed and haven't seen any issues. All tests in the test suite pass.

**Comment by curquiza** [comment]
We should test multiple versions of Node.js in our CI

**Comment by sanders41** [comment]
Currently CI is using Node 12. Node 14 is the current active LTS version so maybe worth adding that one also if you plan to add multiple versions? Node 14 goes to maintenance mode 2021-Oct-19, but its end of life isn't until 2023-Apr-30.

**Comment by bidoubiwa** [comment]
Yes, we need to test and confirm support for node 16 for the `the 26 October 2021`, but before it is not mandatory. This is a reminder issue

**Comment by bidoubiwa** [comment]
/remind me to check compatibility with node 16 on 01/10/21

**Comment by curquiza** [comment]
/remind me to check compatibility with node 16 on 01/10/21
